backend/src/main/java/com/google/sps/datastore/ParticipantDatastore.java

```diff
@@ -168,4 +172,14 @@ public void removeParticipant(String username) {
               + " cannot be removed because it is not in the datastore.");
     }
   }
+
+  /** Convert list of strings to a string with each element delimited by a comma */
+  public static String convertListToString(List<String> list) {
+    return String.join(",", list);
+  }
+
+  /** Convert a string with each element delimited by a comma to a list of strings */
+  public static List<String> convertStringToList(String str) {
+    return Arrays.asList(str.split(","));
+  }
```

Comment:
private

Reply:
I use these methods in ParticipantDatastoreTest. Should I add these methods to the test to make these private?

Reply:
That sounds good. This functionality shouldn't be part of the public API.